src/common/components/Tooltip/Fact/index.js

```diff
@@ -28,13 +36,25 @@ type FactProps = {
     duration: string,
     status: string,
     description: string,
+    apply_count: number,
+    hit_count: number,
+    dmg_multiplier: number,
     prefix: {
       status: string,
       icon: string,
     },
   },
 };
 
+const ATTRIBUTE_MAPPING = {
+  ConditionDamage: 'Condition Damage',
```

Comment:
this all the mapping we need? can you also apply translations here please :)

```
import T from 'i18n-react';

T.translate('itemAttributes.conditionDamage');
```

etc